dtfs-central-api/src/services/state-machines/fee-record/event-handlers/helpers/calculate-fixed-fee.test.ts

```diff
@@ -16,59 +13,30 @@ describe('calculateFixedFeeAdjustment', () => {
     jest.resetAllMocks();
   });
 
-  describe('when the report period starts before the cover start date', () => {
-    it('returns the product of the utilisation, interest percentage (divided by 100), bank admin fee percentage (fixed at 0.9) and number of days remaining in the cover period divided by the day count basis and rounds to 2 decimal places', () => {
-      // Arrange
-      const utilisation = 100000;
-      const interestPercentage = 5;
-      const dayCountBasis = 365;
+  it('returns the product of the utilisation, interest percentage (divided by 100), bank admin fee percentage (fixed at 0.9) and number of days remaining in the cover period divided by the day count basis and rounds to 2 decimal places', () => {
```

Comment:
Could we simplify this, I don't think we need the exact description - if the calculation changes, it'd be easy for this to become out of sync. Maybe something like this instead:

```ts
it('should return the correct calculation'
```